Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -476,12 +495,123 @@ void MemoryWidget::OnSearchAddress()
   m_search_offset->setPalette(offset_palette);
 }
 
-void MemoryWidget::ValidateSearchValue()
+void MemoryWidget::ValidateAndPreviewInputValue()
 {
+  m_data_preview->clear();
+  QString input_text = m_data_edit->text();
+
+  if (input_text.isEmpty())
+    return;
+
+  // Remove any spaces
+  if (!m_input_ascii->isChecked())
+    input_text = input_text.replace(QStringLiteral(" "), QStringLiteral(""));
```

Comment:
Missed this in the initial review (sorry), but this can be simplified to:

```cpp
input_text.remove(QLatin1Char(' '));
```

Reply:
Thanks. The empty one can't be changed I think.

Reply:
I don't get what you mean.

the above is the same as calling

```cpp
input_text.replace(QLatin1Char('_'), "", Qt::CaseSensitive);
```

see [here](https://doc.qt.io/qt-5/qstring.html#remove-1)

Reply:
Sorry, I mean "" is throwing an error for me, unless I QStringLiteral("")

Reply:
lioncash means that you can replace the entire line with just `input_text.remove(QLatin1Char(' '));`, so the whole snippet becomes:

~~~
   if (!m_input_ascii->isChecked())
    input_text.remove(QLatin1Char(' '));
~~~

Reply:
Woo, I'm blind. Thanks for clearing that up!